java-sdk/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 allprojects {
-    version = '0.7.2'
+    version = '0.7.3-SNAPSHOT'
```

Comment:
please use release version

Reply:
Sorry wrong branch. Updated in a separate PR.